packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoPageHeader/WfoHamburgerMenu.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+
+import { signOut } from 'next-auth/react';
+
+import { EuiButtonIcon, EuiContextMenu, EuiPopover } from '@elastic/eui';
+import type { EuiContextMenuPanelDescriptor } from '@elastic/eui';
+
+import { useOrchestratorTheme } from '@/hooks';
+import { WfoLogoutIcon } from '@/icons';
+
+export const WfoHamburgerMenu = ({}) => {
+    const [isPopoverOpen, setPopoverIsOpen] = useState(false);
+    const { theme, isDarkThemeActive } = useOrchestratorTheme();
+
+    const closePopover = () => {
+        setPopoverIsOpen(false);
+    };
+
+    const panels: EuiContextMenuPanelDescriptor[] = [
+        {
+            id: 0,
+            items: [
+                {
+                    name: 'Logout',
+                    icon: (
+                        <WfoLogoutIcon
+                            color={isDarkThemeActive ? 'white' : 'black'}
+                        />
+                    ),
+                    onClick: () => signOut(),
+                },
+            ],
+        },
+    ];
+
+    return (
+        <EuiPopover
+            id={'contextMenuPopoverId'}
+            button={
+                <EuiButtonIcon
+                    iconType="menu"
+                    css={{ color: 'white' }}
```

Comment:
Do use a color from the theme

Reply:
There is no white in the theme. The themeToggle button uses 'white' aswell

Reply:
The color `ghost` is a constant "always white" color. It is preferred to avoid magic values or even a const color like this. (to discuss)